.github/scripts/fetch-base-incrementally.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+# Incrementally fetch git history until we find the merge base with the target branch
+# Usage: ./fetch-base-incrementally.sh <base-ref>
```

Comment:
hmmm.. can't we just `git checkout target-branch` and revert back to current branch instead and it'll download everything we need?

Reply:
see https://github.com/Layr-Labs/eigenda/pull/1817#discussion_r2245581092. Script is gone